src/app/components/LanguageSources.svelte

```diff
@@ -18,6 +18,13 @@
    * @return {void}
    */
   const handleDelete = (): void => {};
+
+  const dict = {
```

Comment:
This constant could have a more descriptive name. How about `englishNameOf`?

```suggestion
  const englishNameOf = {
```

Feel free to come up with a different name for this, but `dict` is too generic!